modules/bullet/collision_object_bullet.h

```diff
@@ -124,12 +127,20 @@ class CollisionObjectBullet : public RIDBullet {
 
 	VSet<RID> exceptions;
 
+	bool needs_body_reload = true;
```

Comment:
Could you initialize `needs_collision_filters_reload`, `needs_body_reload` and `need_shape_reload` to false and let calls to specific methods set these flags when needed, in order to keep things consistent with the update list in space?

Reply:
Good catch! Thanks.